ui/appui-react/src/appui-react/safearea/SafeAreaInsets.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Utilities
+ */
+
+import { SafeAreaInsets as LayoutSafeAreaInsets } from "@itwin/appui-layout-react";
+
+// Re-export `SafeAreaInsets` enum in a non-breaking way to deprecate it in an `appui-layout-react` package.
+
+/** Describes available safe area insets.
+ * @beta
+ */
+export type SafeAreaInsets = LayoutSafeAreaInsets; // eslint-disable-line deprecation/deprecation
+
+/** Describes available safe area insets.
+ * @beta
+ */
+export const SafeAreaInsets = LayoutSafeAreaInsets; // eslint-disable-line @typescript-eslint/no-redeclare, deprecation/deprecation
```

Comment:
What does it mean to have both type and const export ? Isn't `const` enough ?

Reply:
That is needed to correctly re-export both **type** and **value** information of an `enum`:
```
enum A {
  x = 1,
  y = 2,
}

export type B = A;
const b: B = B.x; // 'B' only refers to a type, but is being used as a value here.

export const C = A;
const c: C = C.x; // 'C' refers to a value, but is being used as a type here.

export type D = A;
export const D = A;
let d: D = D.x;
d = A.y;

let a: A = A.x;
a = B.y; // 'B' only refers to a type, but is being used as a value here
a = C.y;
a = D.y;
```

It's the only way I've found to correctly re-export the **enum** w/o any _breaking_ side-effects.
I.e. regular re-export has the same deprecation tag and the linter complains on the consumer side even after importing from `appui-react`:
```
export { SafeAreaInsets } from "@itwin/appui-layout-react";
```